packages/fxa-auth-server/scripts/gen_keys.js

```diff
@@ -40,7 +40,9 @@ try {
   const keysExist = fs.existsSync(pubKeyFile) && fs.existsSync(secretKeyFile);
   assert(!keysExist, 'keys already exists');
 } catch (e) {
-  process.exit();
+  console.log('keys already exist!');
```

Comment:
Will this only throw if the keys exist? I think we can probably remove the comment out code at bottom as well.

Reply:
Ah yes, this was the result of me debugging CI behavior locally. It will just die if the key exists. But of course if you are trying to reproduce an issue locally that can be problematic, because the key likely exists. 

I can either go back to the old behavior, or clean up this new behavior. I wasn't quite sure why it would fail this way. There might be a good good reason behind it. Anyways, let me know what you think is best.

Reply:
I just went rolled back to the previous state 👍